Jenkinsfile

```diff
@@ -25,6 +25,14 @@ pipeline {
         TUNA_ROCM_VERSION = '4.5'
     } 
     stages {
+        stage("solver analytics test") {
+        agent{  label utils.rocmnode("tunatest") }
+        steps {
+          script {
+            utils.solverAnalyticsTest()
+            }
+            }
+        }
```

Comment:
Maybe add "solver analytics test" after the 'code format' and 'pylint' stage? Normally, we would like to check the code format and style first, then do the other check in the CI.

Reply:
I'm trying to test out the solver analytics tests, that's why I moved it to the first stage. Once it's working, I'll move it to the end